src/common/components/kbase_icons.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: "kbase-icons";
+  src:url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.eot");
+  src:url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.eot?#iefix") format("embedded-opentype"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.woff") format("woff"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.ttf") format("truetype"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.svg#kbase-icons") format("svg");
+  font-weight: normal;
+  font-style: normal;
+}
+
+[data-icon]:before {
+  font-family: "kbase-icons" !important;
+  content: attr(data-icon);
+  font-style: normal !important;
+  font-weight: normal !important;
+  font-variant: normal !important;
+  text-transform: none !important;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+[class^="icon-"]:before,
```

Comment:
global css sort of freaks me out. maybe we should have a kbaseIcon component so this can be a css module? thoughts?

Reply:
This was pulled in from navigator. I tried importing it first into the css module but it wasn't working for some reason. I can try again